homeassistant/components/braviatv/entity.py

```diff
@@ -12,23 +12,17 @@ class BraviaTVEntity(CoordinatorEntity[BraviaTVCoordinator]):
 
     _attr_has_entity_name = True
 
-    def __init__(
-        self,
-        coordinator: BraviaTVCoordinator,
-        unique_id: str,
-        model: str,
-    ) -> None:
+    def __init__(self, coordinator: BraviaTVCoordinator, unique_id: str) -> None:
         """Initialize the entity."""
         super().__init__(coordinator)
 
         self._attr_unique_id = unique_id
         self._attr_device_info = DeviceInfo(
             identifiers={(DOMAIN, unique_id)},
+            connections={(CONNECTION_NETWORK_MAC, coordinator.system_info["macAddr"])},
             manufacturer=ATTR_MANUFACTURER,
-            model=model,
-            name=f"{ATTR_MANUFACTURER} {model}",
+            name=coordinator.system_info["name"],
+            model_id=coordinator.system_info["model"],
```

Comment:
This is under the assumption that the model is a real model_id, and not a model name

Reply:
We can also pick to just remove the name from the device info and have the device name take the config entry title

Reply:
> We can also pick to just remove the name from the device info and have the device name take the config entry title

Sounds good

Reply:
Or maybe the name should be `f"{system_info['name']} {system_info['model']}"`?

<img width="575" height="465" alt="obraz" src="https://github.com/user-attachments/assets/c86faad4-b0f8-4a3f-8464-23e8cda452d0" />